modules/indexer/issues/internal/model.go

```diff
@@ -30,7 +30,7 @@ type IndexerData struct {
 	LabelIDs           []int64            `json:"label_ids"`
 	NoLabel            bool               `json:"no_label"` // True if LabelIDs is empty
 	MilestoneID        int64              `json:"milestone_id"`
-	ProjectID          int64              `json:"project_id"`
+	ProjectIDs         []int64            `json:"project_id"`
```

Comment:
I do not think the change is right for indexers, I am pretty sure it breaks existing indexers and the JSON field name doesn't match.